### PR TITLE
Update correct command at step 19 of etcd restore doc for OCP 4.11

### PR DESCRIPTION
**Pull request by alokstech**
Update correct command at step 19

Version(s):

4.11

Issue:

https://issues.redhat.com/browse/OCPBUGS-27437

Link to docs preview:

https://docs.openshift.com/container-platform/4.11/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mramendi** [comment]
Preview link: https://70609--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state step 19

File: modules/dr-restoring-cluster-state.adoc [hunk 1]
```diff
@@ -561,7 +561,7 @@ When the etcd cluster Operator performs a redeployment, the existing nodes are s
 +
 [source,terminal]
 ----
-$ oc patch etcd/cluster --type=merge -p '\{"spec": {"unsupportedConfigOverrides": null}}
```
Comment (JoeAldinger) [inline modules/dr-restoring-cluster-state.adoc:564]: @jinyunma could you PTAL for QE.

Comment (geliu2016) [reply modules/dr-restoring-cluster-state.adoc:564]: /lgtm
GOOD CATCH

Comment (JoeAldinger) [reply modules/dr-restoring-cluster-state.adoc:564]: @geliu2016 should this go into 4.11 only or 4.11-4.15?

Comment (alokstech) [reply modules/dr-restoring-cluster-state.adoc:564]: It's only for 4.11, other docs are okay

Comment (geliu2016) [reply modules/dr-restoring-cluster-state.adoc:564]: Yes, only affect 4.11.

**Comment by JoeAldinger** [comment]
@alokstech thank you for contributing to our docs and @mramendi and @geliu2016 thank you for assisting me with this PR.

**Pull request merged**